### PR TITLE
make the changes in PI file

### PR DESCRIPTION
**Pull request by tlokare**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/DIPU PI/PI.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/DIPU PI/PI.md:1]: The entry is missing an example section showing Java code with `Math.PI` in use.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@tlokare I did a first review and left a few comments.

**Comment by SSwiniarski** [comment]
@tlokare It's been a while. Are you still working on this entry? Do you need assistance implementing any of the suggestions above?

**Comment by caupolicandiaz** [comment]
@tlokare @SSwiniarski I've gone ahead and made the updates. Let me know if further edits are needed.

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz, @tlokare Looks good.

File: content/java/concepts/math-methods/terms/pi/pi.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Example
+
+The following code demonstrates a basic use of the `Math.PI` constant.
+
+```java
+public class Main {
+  public static void main(String[] args) {
+
+    double circle_radius = 5;
+    double circle_area = Math.PI * (circle_radius * circle_radius)
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/pi/pi.md:31]:
```suggestion
    double circle_area = Math.PI * (circle_radius * circle_radius);
```

**Comment by yangc95** [comment]
@tlokare After further eval, I won't be moving forward with a second review. We're trying to keep the terms table within the scope of math methods specifically.

Your idea here will be considered for a future issue to mention math constants that are available here! Thanks so much for your understanding and for creating this PR initially :)

**Comment by yangc95** [comment]
@tlokare I've created an issue for this change here: https://github.com/Codecademy/docs/issues/3601

Feel free to grab it if no one has claimed it. Thank you!

**Pull request closed without merging**